include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -604,7 +608,8 @@ class argument_parser
                  detail::format_ctd*/> format{detail::format_help(0)};
 
     //!\brief List of option/flag identifiers that are already used.
-    std::set<std::string> used_option_ids;
+    std::set<std::string> used_option_ids{"h", "hh", "help", "advanced-help", "export-help", "version"};
+
```

Comment:
Yes, it does mix them. Also, the ids in the add_option calls are appended to this one list... 
I see that this introduces the problem that having `-i` and `--i` simultaneously is not possible as it will through a `parser_design_error` that identifier  "i" was specified before.

Reply:
We also do not require long_ids to be longer than one char... should we ? `--i` seems wierd

Reply:
I think if we require `long_id` to be longer than one character, that would be fine. And it would solve the issue with the IDs being appended to the same list.

Reply:
We could add the check into the id_exists function for long_ids. And then maybe it also makes more sense to rename `id_exists` to `id_valid`?

Reply:
Just kidding, I guess it's better to create a separate if statement to check for the length, so that we can output a relevant throw message.